### PR TITLE
Update attestations.md

### PR DESCRIPTION
**Pull request by lakshya8066**
Capitalizing the word to maintain consistency.

### CODE REVIEW

**Review by MarkLodato — CHANGES_REQUESTED** [review]
Thanks! Would you mind amending your commit to add a Signed-off-by indicating that you agree to the [DCO](https://developercertificate.org/)? (This will make the "DCO" check pass.)

```
$ git commit --amend --no-edit --signoff
```

**Comment by lakshya8066** [comment]
Yeah, sure! I'll do that

**Comment by MarkLodato** [comment]
Looks like you accidentally included 13b5584. Could you please rebase your commit (ce9d3d7) and re-push?

**Comment by lakshya8066** [comment]
Sorry about that let me try again 😅

**Comment by joshuagl** [comment]
Hi @lakshya8066, this change looks good. Can you rebase so that it only includes your change? Then we can merge.

If you `git fetch && git rebase origin/main` (assuming origin is the name of your git remote for the upstream slsa repo) on your branch you'll get a warning about:
> warning: skipped previously applied commit 13b5584

but that's OK – you should have a branch with only your change on top of the commits in main. At which point you can force-push to the branch on GitHub (`git push -f`) and we can merge. 👍

**Review by joshuagl — APPROVED** [review]
Thank you.

**Review by MarkLodato — APPROVED** [review]
